pkg/inject/script.go

```diff
@@ -0,0 +1,60 @@
+package inject
+
+import (
+	"path"
+	"strconv"
+	"strings"
+
+	"github.com/loft-sh/devpod/pkg/template"
+)
+
+func GenerateScript(script string, params *Params, config *Config) (string, error) {
+	rawCode, err := template.FillTemplate(script, map[string]string{
+		"Command":         params.Command,
+		"ExistsCheck":     config.ExistsCheck,
+		"InstallDir":      params.InstallDir(),
+		"InstallFilename": params.InstallFilename(),
+		"PreferDownload":  config.PreferDownload(),
+		"ChmodPath":       config.ChmodPath(),
+		"DownloadBase":    params.DownloadURLs.Base,
+		"DownloadAmd":     params.DownloadURLs.Amd,
+		"DownloadArm":     params.DownloadURLs.Arm,
+	})
+	if err != nil {
+		return "", err
+	}
+
+	return stripCarriageReturns(rawCode), nil
+}
+
```

Comment:
can we merge params and config into one struct or is there a reason for having two?